tools/wpt/browser.py

```diff
@@ -531,39 +531,42 @@ class ChromeChromiumBase(Browser):
         "Darwin": "Mac",
     }.get(uname[0])
 
-    def _get_latest_chromium_revision(self, architecture):
+    def _build_snapshots_url(self, revision, filename):
+        return ("https://storage.googleapis.com/chromium-browser-snapshots/"
+                f"{self._chromium_platform_string}/{revision}/{filename}")
+
+    def _get_latest_chromium_revision(self):
         """Queries Chromium Snapshots and returns the latest Chromium revision number
         for the current platform.
         """
         revision_url = ("https://storage.googleapis.com/chromium-browser-snapshots/"
-                        f"{architecture}/LAST_CHANGE")
+                        f"{self._chromium_platform_string}/LAST_CHANGE")
         return get(revision_url).text.strip()
 
-    def _get_chromium_download_url(self, version=None):
+    def _get_chromium_revision(self, filename, version=None):
         """Format a Chromium Snapshots URL to download a browser component."""
```

Comment:
This documentation needs to be updated, this no longer returns a URL.

Reply:
I hit the merge button too quickly and missed this comment - I'll make the change in the revision pinning PR 😅